Dashboard/app/js/lib/components/devices/AssignmentsEditView/index.jsx

```diff
@@ -0,0 +1,210 @@
+            <div className="formLeftPanel">
+              <fieldset id="surveySelect" className="floats-in">
+                <h2>
+                  02.
+                  {' '}
+                  {strings.selectSurvey}
+                  :
+                </h2>
+                <p className="infoText">{strings.cantFindYourSurvey}</p>
+
+                <div className="SelectLayout">
+                  <label htmlFor="surveyGroup">
+                    {strings.selectSurvey}
+                    :
+                  </label>
+
+                  <FolderSurveySelectorView
+                    initialSurveyGroup={data.initialSurveyGroup}
+                    surveyGroups={data.surveyGroups}
+                    onSelectSurvey={actions.handleSurveySelect}
+                    strings={strings}
+                  />
+                </div>
+
+                <div className="formSelectorList">
+                  <label htmlFor="surveys">
+                    {strings.selectForms}
+                    :
+                  </label>
+
+                  <FormSelectorView
+                    forms={data.forms}
+                    onCheck={actions.handleFormCheck}
+                  />
+                </div>
+              </fieldset>
+            </div>
+
+            <div className="formRightPanel">
+              <fieldset id="surveySelect" className="floats-in">
+                <h2>
+                  03.
+                  {' '}
+                  {strings.selectDevices}
+                  :
+                </h2>
+                <label htmlFor="deviceGroup">{strings.selectDeviceGroup}</label>
+                <DeviceGroupSelectorView
+                  deviceGroupNames={data.deviceGroupNames}
+                  deviceGroups={data.deviceGroups}
+                  deviceGroupIsActive={data.deviceGroupIsActive}
+                  handleDeviceCheck={actions.handleDeviceCheck}
+                />
+              </fieldset>
+            </div>
+          </div>
+
+          <div className="fieldSetWrap makeWhite">
+            <div className="formLeftPanel" />
+            <div className="formRightPanel">
+              {/* Data points list will come here */}
+            </div>
+          </div>
+
+          <div className="menuConfirm">
+            <ul>
+              {showSubmitBtn ? (
+                <li>
+                  <button
+                    onClick={this.onSubmit}
+                    onKeyPress={this.onSubmit}
+                    className="standardBtn"
+                    type="button"
+                  >
+                    {strings.saveAssignment}
+                  </button>
+                </li>
+              ) : (
+                  <li>
+                    <button type="button" className="button noChanges" id="standardBtn">
+                      {strings.saveAssignment}
+                    </button>
+                  </li>
+                )}
+              <li>
```

Comment:
This is the part that wasn't clear. The `onClick` and other actions are only for when the name, and dates values are empty?